server/src/drizzle/methods/User.ts

```diff
@@ -1,118 +1,76 @@
-import { User as TUser } from "../../db/schema";
 import * as jwt from 'jsonwebtoken'
```

Comment:
I believe we could use hono/jwt like we did somewhere else in prisma. But not super important if this is working.